README.md

```diff
@@ -1,64 +1,13 @@
-## launch-ros-sandbox
+# launch-ros-sandbox
 
 Sandboxing plugin for launch_ros
 
-## License
+[Link to documentation]
 
-This library is licensed under the Apache 2.0 License.
-
-## About
```

Comment:
Why was this and all the following sections removed from the README? It was useful information.

Reply:
Because it's been moved to  `index.rst`. Anas, could you maybe simplify index.rst for now and revert the change in the README? I think we can move this content later. I'd like to get this merged sooner than later if we can.

Reply:
> Because it's been moved to `index.rst`. Anas, could you maybe simplify index.rst for now and revert the change in the README? I think we can move this content later. I'd like to get this merged sooner than later if we can.

Seems reasonable to merge when the link https://launch-ros-sandbox.readthedocs.io is populated :-)